### ISSUE TITLE
DevPod incorrectly reports stdout as errors

### ISSUE THREAD

**Issue by nrontsis** [loft-sh/devpod#1145]
**What happened?**  
I got output belonging to stdout as "errors"

**What did you expect to happen instead?**  
Instead of the following output
```
% devpod up minimal-example
13:15:27 info Creating devcontainer...
13:15:27 info Inspecting image mcr.microsoft.com/vscode/devcontainers/python
13:15:27 info a8e8a369f917aa05b34e84a115077384c4c63d9c09cd12904d0c7672003966f9
13:15:28 info Setup container...
13:15:29 info Chown workspace...
13:15:29 info Run command : pipx install uv...
13:15:29 error creating virtual environment...
13:15:29 error installing uv...
13:15:30 error done! ✨ 🌟 ✨
13:15:30 info installed package uv 0.2.18, installed using Python 3.12.3
13:15:30 info These apps are now globally available
13:15:30 info - uv
13:15:30 done Successfully ran command : pipx install uv
13:15:30 info Install extensions 'dbaeumer.vscode-eslint,ms-python.python,ms-python.vscode-pylance,ms-python.autopep8' in the background
13:15:30 info Run 'ssh minimal-example.devpod' to ssh into the devcontainer
13:15:30 info Starting VSCode...
```
I am expecting this one:
```
% devpod up minimal-example
[...]
13:15:29 info creating virtual environment...
13:15:29 info installing uv...
13:15:30 info done! ✨ 🌟 ✨
[...]
``` 

**How can we reproduce the bug?**

In a directory named `minimal-example` with just this file: `.devcontainer/devcontainer.json`:
```
{
  "name": "Development",
  "image": "mcr.microsoft.com/vscode/devcontainers/python",
  "remoteUser": "vscode",
  "postCreateCommand": "pipx install uv"
}
```
run:
```
devpod up ./minimal-example
```

**Local Environment:**  
- DevPod Version: v0.5.16
- Operating System: mac
- ARCH of the OS: ARM64

**DevPod Provider:**  
Tested both in Local provider: docker and Kubernetes Provider (`kubectl version`: v1.29.4).

**Anything else we need to know?**  
Nope

**Comment by mgolisch** [comment]
Its really pipx`s fault. it just prints those lines on stderr
[mgolisch@desktop-michael ~]$ pipx install uv >/dev/null
creating virtual environment...
installing uv...
done! ✨ 🌟 ✨

**Comment by janekbaraniewski** [comment]
Hey @nrontsis thanks for reporting this!
Using stderr for warning or progress logs is a common practice and we've been treating all logs from stderr in our postCreateCommands as error logs. 

To fix that in #1149 we've added parsing of streams of logs from those commands and now all logs from stderr that are not actual errors will be logged by devpod as warnings. So your case will now look like this which hopefully is a bit less confusing:

```
15:17:01 info Creating devcontainer...
15:17:01 info Deleting devcontainer...
15:17:02 info Inspecting image mcr.microsoft.com/vscode/devcontainers/python
15:17:02 info 9c2e1cf2064e71dc36e7efe2ba99c3983766e725dd32ddf4313bfa0dd45d9905
15:17:02 info Setup container...
15:17:02 info Chown workspace...
15:17:02 info Run command : pipx install uv...
15:17:02 warn creating virtual environment...
15:17:03 warn installing uv...
15:17:04 warn done! ✨ 🌟 ✨
15:17:04 info installed package uv 0.2.22, installed using Python 3.12.3
15:17:04 info These apps are now globally available
15:17:04 info - uv
15:17:04 info - uvx
15:17:04 done Successfully ran command : pipx install uv
15:17:04 info Install extensions 'dbaeumer.vscode-eslint,ms-python.python,ms-python.vscode-pylance,ms-python.autopep8' in the background
15:17:04 info Run 'ssh minimal-example.devpod' to ssh into the devcontainer
15:17:04 info Starting VSCode...
```

**Comment by nrontsis** [comment]
Thanks a lot!
